contracts/TopShot.cdc

```diff
@@ -1683,6 +1707,62 @@ pub contract TopShot: NonFungibleToken {
         }
     }
 
+    //------------------------------------------------------------
+    // Contract MetadataViews
+    //------------------------------------------------------------
+
+    /// Return the metadata view types available for this contract
+    ///
+    view access(all) fun getContractViews(resourceType: Type?): [Type] {
```

Comment:
I'd recommend implementing `Royalties` here too since it looks like it is the same for all moments. Then you can just call `resolveContractView` for royalties from the `NFT` one to save code